snips_helper/intent.py

```diff
@@ -24,9 +24,27 @@ def utterances(self):
         self.__deactivate()
         return training_data
 
+    def import_utterances(self, utterances):
+        print("Importing utterances")
+        self.__activate()
+        self.__get_utterance_dot_button().click()
+        self.__get_import_utterance_button().click()
+        self._get_modal_text_area().send_keys(utterances)
+        self.__get_modal_import_button().click()
+        time.sleep(1)
+        self.save()
+        return
+
+    def delete_utterances(self):
+        print("Importing utterances")
+        self.__activate()
+        while self.__get_utterance_delete_button():
+            self.__get_utterance_delete_button().click()
+        self.save()
+        return
+
     def save(self):
         print("Saving intent: {}".format(self.name))
-        self.__activate()
```

Comment:
instead of removing this, can we add a check to see if the save button is present, and if so, skip activating.  This way we can use this for both for re-training of the assistance and saving the intent after an edit of the intent.

Reply:
I think the button is always there, but isn't clickable so added that check.

Reply:
NVM I see what you mean.